### DISCUSSION TITLE
关闭了评论，让一切简单起来

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#112] [Announcements]
https://bbing.com.cn/202205/disbalecomment-makeitsimple/

**Comment by penndu** [comment]
然后又屈服了？
github评论工具确实可以过滤掉一些评论。
毕竟不是所有人都有github账号。
且github在某些地区不可用~

**Comment by penndu** [comment]
gitalk有一年多没有更新了，有时候刷新不出来评论。
推荐下Beaudar，也是基于GitHub的评论系统，可无缝隙切换！

**Comment by caibingcheng** [comment]
> gitalk有一年多没有更新了，有时候刷新不出来评论。
> 推荐下Beaudar，也是基于GitHub的评论系统，可无缝隙切换！

Beaudar也是通过GitHub api访问的，网络稳定性应该和gitalk差不多吧，我现在gitalk有些请求是走的vercel代理的，改天看看能不能全走vercel，这样网络可能好点…（但是哪天要是vercel没了，那又得搞好一会了）

**Comment by penndu** [comment]
> @caibingcheng
> > gitalk有一年多没有更新了，有时候刷新不出来评论。
> > 推荐下Beaudar，也是基于GitHub的评论系统，可无缝隙切换！
> 
> Beaudar也是通过GitHub api访问的，网络稳定性应该和gitalk差不多吧，我现在gitalk有些请求是走的vercel代理的，改天看看能不能全走vercel，这样网络可能好点…（但是哪天要是vercel没了，那又得搞好一会了）

关键是我挂着梯子，访问还是会出现不稳定~

**Comment by caibingcheng** [comment]
> @penndu
> > @caibingcheng
> > > gitalk有一年多没有更新了，有时候刷新不出来评论。
> > > 推荐下Beaudar，也是基于GitHub的评论系统，可无缝隙切换！
> > 
> > Beaudar也是通过GitHub api访问的，网络稳定性应该和gitalk差不多吧，我现在gitalk有些请求是走的vercel代理的，改天看看能不能全走vercel，这样网络可能好点…（但是哪天要是vercel没了，那又得搞好一会了）
> 
> 关键是我挂着梯子，访问还是会出现不稳定~

请问你遇到访问不稳定的时候, 是通过issue的链接点进来的吗? 现在发现通过带code参数的链接[https://bbing.com.cn/202205/disbalecomment-makeitsimple/?code=123](https://bbing.com.cn/202205/disbalecomment-makeitsimple/?code=123)访问, gitalk就会加载不出来

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > > @caibingcheng
> > > > gitalk有一年多没有更新了，有时候刷新不出来评论。
> > > > 推荐下Beaudar，也是基于GitHub的评论系统，可无缝隙切换！
> > > 
> > > Beaudar也是通过GitHub api访问的，网络稳定性应该和gitalk差不多吧，我现在gitalk有些请求是走的vercel代理的，改天看看能不能全走vercel，这样网络可能好点…（但是哪天要是vercel没了，那又得搞好一会了）
> > 
> > 关键是我挂着梯子，访问还是会出现不稳定~
> 
> 请问你遇到访问不稳定的时候, 是通过issue的链接点进来的吗? 现在发现通过带code参数的链接[https://bbing.com.cn/202205/disbalecomment-makeitsimple/?code=123](https://bbing.com.cn/202205/disbalecomment-makeitsimple/?code=123)访问, gitalk就会加载不出来

收到回复时会有邮件通知的，我是通过邮件通知链接点进来的！

**Comment by goodpaperman** [comment]
精神内耗有什么表现
